src/components/UserGroup.tsx

```diff
@@ -22,18 +22,69 @@ const UserImage = styled(UserPic)`
     display: block;
 `;
 
+const StyledCounter = styled.div`
+    height: 23px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-inline: 7px;
+    background-color: ${gray4};
+    position: relative;
+    right: 10px;
+    top: 1px;
+`;
+const StyledSmallCircle = styled.div`
+    width: 13px;
+    height: 13px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${gray6};
+    position: relative;
+    right: 17px;
+    top: 7px;
+`;
+
+const StyledText = styled(Text)`
+    position: absolute;
+    bottom: 58%;
```

Comment:
What the numbers? Can u explain the logic?

Reply:
``` css
display: flex;
align-items: center;
```

?

Reply:
this solution is not suitable in this case